astropy/units/tests/test_quantity.py

```diff
@@ -733,7 +737,10 @@ def test_scalar_quantity_str(self):
 
     def test_scalar_quantity_repr(self):
         assert repr(self.scalarintq) == "<Quantity 1 m>"
-        assert repr(self.scalarfloatq) == "<Quantity 1.3 m>"
+        if NUMPY_LT_1_14:
+            assert repr(self.scalarfloatq) == "<Quantity 1.3 m>"
+        else:
+            assert repr(self.scalarfloatq) == "<Quantity  1.3 m>"
```

Comment:
Huh. I know you're probably just making changes to make it consistent with what numpy does, but seems strange that an extra space is added here?